src/components/general/world-objects-list/WorldObjectsList.jsx

```diff
@@ -139,39 +109,66 @@ export const WorldObjectsList = ({ opened, setOpened }) => {
 
     };
 
-    const handleOnFocusLost = ( event ) => {
+    const handleBackBtn = () => {
 
-        event = event ?? {};
+        setSelectedApp( null );
 
-        if ( event.detail && event.detail.dispatcher === componentName ) return;
+    };
 
-        if ( ! event.detail ) {
+    //
 
-            ioManager.click( new MouseEvent('click') );
-            cameraManager.requestPointerLock();
+    useEffect( () => {
 
-        }
+        AppUIStateManager.state.world = isOpened;
```

Comment:
Not a good idea to use `AppUIStateManager` IMO.

It's especially bad to add global state and add key handing to it, since that is doubling state with the engine, creating multiple sources of truth.